Rakefile

```diff
@@ -13,8 +13,6 @@ Rake::TestTask.new(:test) do |t|
   t.libs << 'lib'
   t.libs << 'test'
   t.pattern = ENV['TEST_PATTERN'] || 'test/**/*_test.rb'
-  t.verbose = ENV['TEST_VERBOSE'] == '1'
-  t.warning = true
```

Comment:
After checking this again, the warnings are from rails 3.2, etc. Those versions are no longer maintained and won't be patched. So :+1: for this change right now.